docs/spec/api.md

```diff
@@ -618,26 +618,6 @@ The "digest" parameter must be included with the PUT request. Please see the
 [_Completed Upload_](#completed-upload) section for details on the parameters
 and expected responses.
 
-Additionally, the upload can be completed with a single `POST` request to
```

Comment:
I see.

@SvenDowideit Typically, distribution doesn't do cherry picking, as it wrecks history.